FilePersistence/src/version_control/Commit.cpp

```diff
@@ -103,4 +103,60 @@ bool Commit::getFileContent(QString fileName, const char*& content, int& content
 		return false;
 }
 
+QStringList Commit::nodeLinesFromId(Model::NodeIdType Id, bool findChildrenByParentId) const
```

Comment:
please follow the Envision camelCaseRules. Variable names start with a lowercase letter, not upper case `id`.